api/bems.py

```diff
@@ -0,0 +1,33 @@
+# Client
+import socket
+import sys
+import requests
+import json
+from utils.logging import logging
+with open('dr.json', 'r') as f:
+    json_data = json.load(f)
+
+r = requests.post(
```

Comment:
Could you please change the url as a configuration parameter?

Reply:
Fix in 1667759.